src/Host.h

```diff
@@ -250,6 +250,7 @@ class Host : public QObject
     void startSpeedWalk();
     void saveModules(int sync, bool backup = true);
     void reloadModule(const QString& reloadModuleName);
+    QStringList getModulesToSync() { return mModulesToSync; }
```

Comment:
Given that this method is NOT modifying the data of this class, it is probably good coding practice to say so, as:
```cpp
    QStringList getModulesToSync() const { return mModulesToSync; } 
```
if you really want to go the whole hog you can instead return a reference to the original data rather than creating a shallow copy - but since Qt optimises things to avoid creating the copy to return that the above at first glance implies there is not really much point of making it:
```cpp
    const QStringList& getModulesToSync() const { return mModulesToSync; }
```